appveyor.yml

```diff
@@ -0,0 +1,13 @@
+# Add 32bit support soon
```

Comment:
I don't think we need 32bit support on appveyor. The last 32 bit CPU Intel shipped was in 2008. I don't want to care about 32-bit Windows anymore.

Drop the comment.